gatsby-site/src/layouts/index.js

```diff
@@ -40,12 +45,11 @@ const Layout = ({ children, data }) => (
         { name: 'pinterest-rich-pin', content: 'false' }
       ]}
     />
-    <ReactNavbar siteTitle={data.site.siteMetadata.title} />
+    {/* <ReactNavbar siteTitle={data.site.siteMetadata.title} /> */}
```

Comment:
Do we need to keep this? I assume this is the same component that is now in View/index.js